packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -155,6 +156,7 @@ class ArtifactStore {
     return null;
   }
 
+  static String flutterRoot;
```

Comment:
Good idea.  Having these in statics is kind of wonky.  I bet there's a better way to do it.